code-challenges.js

```diff
@@ -13,16 +13,53 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+//this would be the Constant with the perameter 
 
+//const boiling = (temp) =>{
 
+    // Input:(numbers)
+    //Output(strings)
 
+   // This is where i would creat the condition for the function 
+   //if the tempature is less than boiling point then the output will be "below boiling point"
+   //if tempature is equal to boiling point output will be boiling point
+   //if temp is more than boiling point output will be above boiling point 
+// if(temp < 212){
+//     return `${temp} Below boiling point `
 
+// }else if(temp === 212){
+//     return `${temp} at boiling point`
+// }else if(temp > 212){
+//     return `${temp} is over boiling point`
+// }
+
+
+// }
+
+// console.log(boiling(temp1))
+// console.log(boiling(temp2))
+// console.log(boiling(temp3))
 // --------------------2) Create a function that takes in an array of numbers and returns an array with each number multiplied by 5.
 // Use the test variable provided below. Expected output: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
+//var myNumbers1 = [3, 7, 0, 6, -9]
+
+// this will be where i put my constant and my perameter 
+ //const multi = (number1)
+ //this is where the loop would go to make all the numbers multiplied by 5
+ //input:numbers 
+ //output:numbers mulplied by 5
+
+//  let multi2 = []
 
+// for(let i=0; i<array.length; i++){
+//     multi2.push(array[i] * 5)
+//     }
+//     return multi2
+// }
+// console.log(multi(myNumbers1))
 
+// looked through the notes to try and fix it. i kept getting a syntaxerror and for an unexpected toke `}` could not figure out why it didnt work
 
```

Comment:
This looks really close. I think the issue is the function declaration. I see a const on line 48 but it looks like the variable is being passed into the parentheses instead of a parameter and the => and opening { are missing.